src/main/scala/scorex/util/serialization/Reader.scala

```diff
@@ -0,0 +1,63 @@
+package scorex.util.serialization
+
+trait Reader {
+
+  type CH
+
+  def getChunk(size: Int): CH
+  /**
+    * Get a byte at current position without advancing the position.
+    * @return byte at current position
+    */
+  def peekByte(): Byte
+  def getByte(): Byte
```

Comment:
@VictorMikheev please add blank line, otherwise the doc comment is  confusing

Reply:
fixed